openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -829,20 +830,29 @@ class TestAccountDeactivation(TestCase):
 
     def setUp(self):
         super(TestAccountDeactivation, self).setUp()
-        self.superuser = SuperuserFactory()
-        self.staff_user = AdminFactory()
         self.test_user = UserFactory()
         self.url = reverse('accounts_deactivation', kwargs={'username': self.test_user.username})
 
-    def assert_activation_status(self, expected_status=status.HTTP_200_OK, expected_activation_status=False):
+    def build_jwt_headers(self, user):
+        """
+        Helper function for creating headers for the JWT authentication.
+        """
+        expires_in = settings.OAUTH_ID_TOKEN_EXPIRATION
```

Comment:
Also, for future reference, you  don't have to test JWT authentication for every test. Pretty much all of my APIs support session authentication, so I use that. You also have the option of using [`force_authenticate`](http://www.django-rest-framework.org/api-guide/testing/#forcing-authentication). Just an FYI. What you have here is fine.